pkg/controller/controller_binding.go

```diff
@@ -427,6 +431,8 @@ func (c *controller) reconcileServiceBinding(binding *v1beta1.ServiceBinding) er
 			}
 		}
 
+		toUpdate.Status.UnbindStatus = v1beta1.ServiceBindingUnbindStatusRequired
```

Comment:
If you move this either inside the previous if block or before it, then you'll be good. The status update is sent in `recordStartOfServiceBindingOperation`.

Reply:
Moved, thanks!